spanner/sessionclient.go

```diff
@@ -313,6 +313,62 @@ func (sc *sessionClient) executeBatchCreateSessions(client *vkit.Client, createC
 	}
 }
 
+func (sc *sessionClient) executeCreateMultiplexedSessions(ctx context.Context, client *vkit.Client, labels map[string]string, md metadata.MD, consumer sessionConsumer) {
+	ctx = trace.StartSpan(ctx, "cloud.google.com/go/spanner.CreateSession")
```

Comment:
This method largely seems like a copy-paste of `createSession`. Can we combine the two?